### PR TITLE
Comment zipper functions. Fix foldr1

### PR DESCRIPTION
**Pull request by kubaracek**
foldr1 used foldl1, fixed typo

### CODE REVIEW

File: src/List/NonEmpty/Zipper.elm [hunk 1]
```diff
@@ -690,25 +712,48 @@ absoluteIndexedMap f (Zipper p focus n) =
...
+{-| Collapse `Zipper a` into `a` value from left
+
+    foldl1 (++) <| custom ["hello"] " " ["world"]
+    --> "world hello"
+
+-}
 foldl1 : (a -> a -> a) -> Zipper a -> a
 foldl1 f =
-    NE.foldr1 f << toNonEmpty
+    NE.foldl1 f << toNonEmpty
```
Comment (turboMaCk) [inline src/List/NonEmpty/Zipper.elm:734]: uh... good catch.

File: src/List/NonEmpty/Zipper.elm [hunk 2]
```diff
@@ -661,21 +661,43 @@ rewindByHelper step n acc =
 -- Functor
 
 
-{-| -}
+{-| Map a function over Zipper
+
+    map (\x -> x + 1) (custom [1] 2 [3, 4])
+    |> toList
+    --> [2, 3, 4, 5]
```
Comment (turboMaCk) [inline src/List/NonEmpty/Zipper.elm:668]: I'm not sure why `elm/core` uses functions like this as an example. I think it's important to show that type can be changed within the map while this function simply operates as `(a -> a)`.

**Comment by turboMaCk** [comment]
@kubaracek run elm-format over the code to make CI pass `elm-format --yes src`

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+test:
```
Comment (turboMaCk) [inline Makefile:1]:
```suggestion
.PHONY: test
test:
```

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+	elm-verify-examples && elm-test
+
+format:
```
Comment (turboMaCk) [inline Makefile:4]:
```suggestion
.PHONY: format
format:
```

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+	elm-format --yes src
+
+.PHONY: test format
```
Comment (turboMaCk) [inline Makefile:7]:
```suggestion
```

I prefer having `.PHONY` above the target - it's easier to maintain in my opinion.

File: src/List/NonEmpty/Zipper.elm (hunk 2 above)
Comment (kubaracek) [reply src/List/NonEmpty/Zipper.elm]: This was from my head actually :D Added String.fromInt

**Comment by turboMaCk** [comment]
Thanks a lot!

**Pull request merged**